### PR TITLE
Added Favicon And Logo

### PR DESCRIPTION
**Pull request by rossoskull**
ping @aashutoshrathi . PR for issue #2

### CODE REVIEW

**Comment by aashutoshrathi** [comment]
Is this being updated on Pythonanywhere as well?

**Comment by rossoskull** [comment]
I don't have any idea about that

**Comment by Monal5031** [comment]
@aashutoshrathi want to setup auto-deployment?

File: arc/teams/static/arcadia/style.css [hunk 1]
```diff
@@ -72,4 +72,10 @@ body {
 
 .input-field {
 	margin-top: 25px;
+}
+
+.brand-logo img {
+  display: block;
+  height: 50px;
+  margin: 10px 0px;
 }
```
Comment (Monal5031) [inline arc/teams/static/arcadia/style.css:81]: newline

**Review by Monal5031 — CHANGES_REQUESTED** [review]
This PR is for favicon and logo right? then why migration files are being deleted?
Also amend your commit to reflect the changes done properly.

**Comment by rossoskull** [comment]
@Monal5031 The .pyc files change every time I run the server. Also they are not ignored by gitignore. So they are changed automatically. And about the changes in css file, they are for proper positioning and sizing of the logo.

**Pull request merged**